### PR TITLE
Update koKR.lua

### PR DESCRIPTION
**Pull request by dongminrocks**
Back for DF.

Will fix every week!

Thanks!

### CODE REVIEW

File: VaultOfTheIncarnates/Locales/koKR.lua [hunk 1]
```diff
@@ -77,44 +77,44 @@ if L then
...
 
-	-- -- General
-	-- L.primal_attunement = "Soft Enrage"
+	 -- 일반
+	 L.primal_attunement = "미니 광폭"
 
-	-- -- Stage 2
-	-- L.violent_upheaval = "Pillars"
+	 -- Stage 2
+	 L.violent_upheaval = "기둥 "
```
Comment (ntowle) [inline VaultOfTheIncarnates/Locales/koKR.lua:104]: extra space here?

File: VaultOfTheIncarnates/Locales/koKR.lua [hunk 2]
```diff
@@ -1,65 +1,65 @@
...
+	 L.meteor_axes = "도끼" -- Meteor Axes
+	 L.meteor_axe = "도끼" -- Singular
+	 L.meteor_axes_melee = "근딜 도끼"
+	 L.meteor_axes_ranged = "원딜 도끼"
+	 L.conductive_marks = "표식" -- Conductive Marks
+	 L.conductive_mark = "표식" -- Singular
+
+	 L.custom_off_chain_lightning = "연쇄 번개는 기본적으로 꺼져있습니다. 활성화해서 연쇄 번개를 활성화하십시오."
+
+	 L.custom_on_stop_timers = "항상 능력 바 표시Always show ability bars"
```
Comment (ntowle) [inline VaultOfTheIncarnates/Locales/koKR.lua:38]: did you mean to leave the english in?

File: VaultOfTheIncarnates/Locales/koKR.lua [hunk 3]
```diff
@@ -1,65 +1,65 @@
...
-	-- L.custom_on_nameplate_fixate_desc = "Show an icon on the nameplate of Frenzied Tarasek that are fixed on you.\n\nRequires the use of Enemy Nameplates and a supported nameplate addon (KuiNameplates, Plater)."
-
-	-- L.molten_cleave = "Frontal"
-	-- L.incinerating_roar = "Roar"
-	-- L.molten_spikes = "Spikes"
-	-- L.collapsing_army = "Army"
-	-- L.greater_flamerift = "Mythic Add"
-	-- L.leaping_flames = "Flames"
+	 L.custom_on_nameplate_fixate = "시선 고정 네임플레이트 아이콘"
+	 L.custom_on_nameplate_fixate_desc = "당신에게 시선이 고정된 타라세크의 네임플레이트에 아이콘을 표시합니다.\n\n 네임플레이트를 지원하는 애드온(KuiNameplates, Plater)을 사용하고 적 네임플레이트가 활성화된 상태여야함."
```
Comment (ntowle) [inline VaultOfTheIncarnates/Locales/koKR.lua:5]: did you intend to leave the space after "\n\n"? double check how it's displaying in-game

File: VaultOfTheIncarnates/Locales/koKR.lua [hunk 4]
```diff
@@ -1,65 +1,65 @@
...
+
+	 L.molten_cleave = "전방기"
+	 L.incinerating_roar = "포효"
+	 L.molten_spikes = "쐐기"
+	 L.collapsing_army = "쫄 소환"
+	 L.greater_flamerift = "신화 쫄"
+	 L.leaping_flames = "Flames"
```
Comment (ntowle) [inline VaultOfTheIncarnates/Locales/koKR.lua:12]: was this one missed?

File: VaultOfTheIncarnates/Locales/koKR.lua (hunk 2 above)
Comment (dongminrocks) [reply VaultOfTheIncarnates/Locales/koKR.lua:38]: yes, I could translate it but I want to wait for it to go live before I do it

File: VaultOfTheIncarnates/Locales/koKR.lua (hunk 1 above)
Comment (dongminrocks) [reply VaultOfTheIncarnates/Locales/koKR.lua:104]: oops didn't mean for the extra space

File: VaultOfTheIncarnates/Locales/koKR.lua (hunk 2 above)
Comment (dongminrocks) [reply VaultOfTheIncarnates/Locales/koKR.lua:38]: for the other abilities I can look up stuff on wowhead and whatnot but I couldn't really grasp what those really meant

File: VaultOfTheIncarnates/Locales/koKR.lua (hunk 4 above)
Comment (ntowle) [reply VaultOfTheIncarnates/Locales/koKR.lua:12]: Here is the ability: https://www.wowhead.com/spell=394917/leaping-flames

Comment (dongminrocks) [reply VaultOfTheIncarnates/Locales/koKR.lua]: translated, but I'll definitely keep an eye out on- I think I can definitely find a better word for it

**Comment by funkydude** [comment]
Thanks! Keep in mind there's multiple locale files that are missing a few translations also.

**Pull request merged**